CONTRIBUTING.md

```diff
@@ -6,3 +6,8 @@ Thanks for taking the time to contribute!
 
 If you're a maintainer use `npm run release` to start the release
 process and follow the instructions printed to the console.
+
+## Publishing a new chart release
+
+If you're a maintainer use `./release-chart.sh <new-chart-version>` to start the release
```

Comment:
How should someone get the new-chart-version to use?

Reply:
I am thinking we can get the new-chart-version by bumping the chart version here https://github.com/godaddy/kubernetes-external-secrets/blob/master/charts/kubernetes-external-secrets/Chart.yaml#L3 based on the changes. I can add this to the doc if it looks good to you.

Reply:
sgtm